### DISCUSSION TITLE
Descriptions and summaries

### DISCUSSION THREAD

**Discussion by titpetric** [go-task/task#1251] [General]
The taskfile already includes `desc` and `summary` fields at task level. Is there any objection in adding these into the root taskfile schema, to next to `version`? And also extend `task -l` to print the taskfile desc, and with `--summary` to print the taskfile summary as well?

https://taskfile.dev/api/#taskfile-schema

I tried searching for duplicate issues, but it's a bit overloading. I'll gladly open a PR to extend above if there is agreement, and if there's nothing like it in flight :)

**Comment by titpetric** [comment]
A bit more context: Ultimately I want to use `task -l --summary > Taskfile.md` for making people's review lives easier. For that a Taskfile summary and description is welcome.

**Comment by pd93** [comment]
Sounds similar to #1094

**Comment by titpetric** [comment]
This is how I'm patching it currently:

https://github.com/titpetric/task-ui/blob/main/README.md.sh
